extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -127,6 +127,7 @@ export let InteractiveReactData;
  *    consentId: ?string,
  *    currentPageId: string,
  *    currentPageIndex: number,
+ *    pageDistanceMap: !Map<string, !number>,
```

Comment:
```suggestion
 *    pageDistanceMap: !Map<string, number>,
```

primitive types don't need to be marked with the non-nullable (!) by default.